poster/poster.tex

```diff
@@ -0,0 +1,367 @@
+ \item AlexNet~\citep{krizhevsky2012imagenet}: extract features of our data set using the AlexNet model without additional training. 
+ \item SciKit Learn~\citep{scikit-learn}: a Python library of machine learning algorithms. 
+ \begin{itemize}
+ \item Applied C-Support Vector Classification (\emph{sklenar.svm.svc}) to the extracted features for classification. 
+ \item Lin, RBF, sigmoid and polynomial kernels were compared. 
+ \item Parameters optimization through exhaustive grid search (\emph{sklearn.grid\_search}).
+ \end{itemize}
+ \item Each set of four rotated pictures were either in the training or testing set. 
+  \item Workflow of this project is shown as a flowchart in Figure~\ref{fig:flowchart}
+ \end{itemize}
+
+
+  \end{block} %tools and methods block
+
+\end{column}
+
+    \begin{column}{\sepwid}\end{column}			% empty spacer column
+
+
+        \begin{column}{\onecolwid}
+
+          
+ \begin{block}{Workflow}
+
+\tikzstyle{block} = [rectangle, draw, fill=SFUred!20, 
+    text width=30em, text centered, rounded corners, minimum height=2em]
+    \tikzstyle{small} = [rectangle, draw, fill=white!20, 
+    text width=5em, text centered, rounded corners, minimum height=2em]
+        \tikzstyle{medium} = [rectangle, draw, fill=white!20, 
+    text width=8em, text centered, rounded corners, minimum height=2em]
+
+\tikzstyle{line} = [draw, -latex']
+
+\begin{figure}   
+\centering 
+\begin{tikzpicture}[node distance = 4cm, auto]
+    % Place nodes
+    \node [block] (capture) {Capture images};
+    \node [block, below of=capture] (process) {Label and process images (crop, resize and convert to RGB)};
+%    \node [block, below of=process] (convert) {Convert to RGB };
+    \node [block, below of= process] (cnn) {Extract features via CNN (Caffe and AlexNet)};
+    \node [block, below of=cnn] (svm) {Classify data via SVM (SciKit Learn)};
+    \node [medium, below left = 2cm and -12cm of svm] (nonbanana) {Non-Banana};
+    \node [small, below right = 2cm and -14 cm of svm] (banana) {Banana};
+    \node [small, below of = banana] (ripe) {Ripe};
+    \node [small, below left = 1.8 cm and 1.8 cm of banana] (unripe) {Unripe};
+     \node [small, below right = 1.8cm and 1.8 cm of banana] (overripe) {Overripe};
+
+    % Draw edges
+    \path [line] (capture) -- (process);
+    \path [line] (process) -- (cnn);
+%    \path [line] (convert)--(cnn);
+    \path [line] (cnn)--(svm);
+    \path [line] (svm)--(nonbanana);
+        \path [line] (svm)--(banana);
+          \path [line] (banana)--(unripe);
+        \path [line] (banana)--(overripe);
+	\path [line] (banana)--(ripe);
+\end{tikzpicture}
+\caption{A flow chart of project development.}
+\label{fig:flowchart}
+ \end{figure}
+  \end{block} %tools and methods block
+         
+%         \begin{theorem}\label{linearHamiltonianTheorem}
+\vskip1ex
+
+\begin{alertblock}{Features Extraction via AlexNet}
+
+\begin{figure}[h]
+\centering
+\includegraphics[width=\textwidth]{alexnet.png}
```

Comment:
Here's my write up on AlexNet:

Alexnet is a convolutional neural network designed and trained by alex et al to classify images for the ILSVRC-2012 challenge. Alexnet has five convolutional layers, some of which are followed by max-pooling, and then 3 fully connected layers (the first two were trained with dropout, in effect 50% of their weights are 0). A convolutional layer is a neural network layer where each neuron is connected to a small, adjacent portion of neurons on the previous layer. Each neuron on the previous layer is connected to several, adjacent neurons on the next layer. To extract features from our dataset we feed the image through the pre-trained network and extract the internal representation of the image at the last 3 layers (fc6, fc7, and fc8). The internal representation at this layers is either a vector of length 4096 (for fc6 and fc7) or a vector of length 1000 (for fc8). We evaluate classification on the output of each layer.

Let me know if you want to change the format, include other info, clarify something, or change it in anyway.